web-client/iron-remote-desktop/src/services/remote-desktop.service.ts

```diff
@@ -188,13 +188,9 @@ export class RemoteDesktopService {
             .catch((err: IronError) => {
                 this.setVisibility(false);
 
-                this.raiseSessionEvent({
-                    type: SessionEventType.ERROR,
-                    data: err.backtrace(),
-                });
                 this.raiseSessionEvent({
                     type: SessionEventType.TERMINATED,
```

Comment:
question: Shouldn’t we raise an `ERROR`, to get the red toast in this case?

Reply:
Toast will be red for `TERMINATED`.